imageio/imageio-metadata/src/main/java/com/twelvemonkeys/imageio/metadata/tiff/TIFFWriter.java

```diff
@@ -292,7 +292,17 @@ private long writeValue(final Entry entry, final long dataOffset, final ImageOut
 
     private int getCount(final Entry entry) {
         Object value = entry.getValue();
-        return value instanceof String ? ((String) value).getBytes(StandardCharsets.UTF_8).length + 1 : entry.valueCount();
+        if(value instanceof String){
+            return ((String) value).getBytes(StandardCharsets.UTF_8).length + 1;
+        } else if(value instanceof String[]){
+            int sum = 0;
+            for (String string : (String[]) value){
+                sum += string.getBytes(StandardCharsets.UTF_8).length + 1;
+            }
+            return sum;
+        } else {
+            return entry.valueCount();
+        }
```

Comment:
This block has some code style issues (space after if, else on new line, space before opening brace etc).